tools/symlink.js

```diff
@@ -4,7 +4,7 @@ const yargs = require("yargs");
 const { hideBin } = require("yargs/helpers");
 
 /**
- * @typedef {{dataPath: string; symLinkName: string;}} FoundryConfig
+ * @typedef {{dataPath: string; symLinkName: Array<string>;}} FoundryConfig
```

Comment:
these two are switched

`{{dataPath: Array<string>; symLinkName: string;}} FoundryConfig

Reply:
oh, haha just keeping you on the toes ;) but it is fixed in the latest commit!